std/algorithm/mutation.d

```diff
@@ -881,6 +881,52 @@ void move(T)(ref T source, ref T target)
     }
 }
 
+///
+unittest
+{
+    Object obj1 = new Object;
+    Object obj2 = obj1;
+    Object obj3;
+
+    move(obj2, obj3);
```

Comment:
Not sure what this example is supposed to show. I guess it could be useful for programmers fresh out of C++ who don't even know about the class/struct split... but it seems like trivia to put it here. Oh well, it's not a big example anyway.